urllib3/poolmanager.py

```diff
@@ -110,8 +116,14 @@ class PoolManager(RequestMethods):
 
     proxy = None
 
-    def __init__(self, num_pools=10, headers=None, **connection_pool_kw):
+    def __init__(self, num_pools=10, headers=None, transport_security_manager=None,
+                 **connection_pool_kw):
         RequestMethods.__init__(self, headers)
+
+        if transport_security_manager is None:
+            transport_security_manager = TransportSecurityManager()
```

Comment:
The way to turn this off would be to pass a no-op subclass of TransportSecurityManager. Right now TransportSecurityManager is a no-op; in the next iteration I will include something like NullTransportSecurityManager, so:

```
from urllib import PoolManager
from urllib.transport_security import NullTransportSecurityManager

pool = PoolManager(transport_security_manager=NullTransportSecurityManager())
```